tasks/prepare-gitlab-runner.yml

```diff
@@ -8,6 +8,28 @@
   tags:
     - always
 
+- name: Install EPEL repo.
+  yum:
+    name: https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{ ansible_distribution_major_version }}.noarch.rpm
+    state: present
+  when: ansible_distribution == 'CentOS'
+  register: install_epel
+  until: install_epel is succeeded
+  become: true
+  tags:
+    - always
+
+- name: Import EPEL GPG key.
+  rpm_key:
```

Comment:
Please, use dynamic include for OS-specific tasks

Reply:
Done